.travis.yml

```diff
@@ -5,7 +5,7 @@ env:
   - NUMPY_VERSION=stable
   - MAIN_CMD='python setup.py'
   - CONDA_DEPENDENCIES='xarray dask toolz Cython pykdtree sphinx cartopy rasterio pillow matplotlib
-    pyyaml pyproj coveralls configobj coverage codecov pytest pytest-cov shapely'
+    pyyaml pyproj coveralls configobj coverage codecov pytest pytest-cov shapely zarr scipy'
```

Comment:
Why is scipy required? I thought it wasn't needed since we moved to pykdtree for all the KDTree stuff?

Reply:
The legacy Numpy based resampler uses Scipy to query in parallel.